test/cpp/jit/test_gpu.cpp

```diff
@@ -776,15 +776,15 @@ void testGPU_FusionParser() {
     }
   }
 
-  Fusion fusion;
-  FusionGuard fg(&fusion);
   torch::jit::fuser::cuda::CudaKernel prog;
+  Fusion& fusion = *prog.fusion_;
+  FusionGuard fg(&fusion);
```

Comment:
Does it make sense to have a shortcut like below?
```
torch::jit::fuser::cuda::CudaKernel prog;
FusionGuard fg(prog);
```

Reply:
I put a wrapper to construct `FusionGuard` from `CudaKernel` and it is used in kernel.cpp.
But here `fusion` is also used in the `GPULower` down and I think it doesn't hurt to leave fusion here.